src/dbnode/storage/bootstrap/bootstrapper/readers.go

```diff
@@ -206,8 +203,8 @@ func newShardReaders(
 		}
 
 		openOpts := fs.DataReaderOpenOptions{
-			Identifier:                fs.NewFileSetFileIdentifier(ns.ID(), blockStart, shard, info.VolumeIndex),
-			OptimizedReadMetadataOnly: optimizedReadMetadataOnly,
+			Identifier:       fs.NewFileSetFileIdentifier(ns.ID(), blockStart, shard, info.VolumeIndex),
+			StreamingEnabled: optimizedReadMetadataOnly,
```

Comment:
nit rename `optimizedReadMetadataOnly` to `streamingEnabled` here and other places it's used, including `EnqueueReadersOptionsEnqueueReadersOptions`

Reply:
This flag indicates whether we will be reading `data` or `metadata`. Streaming is just the implementation detail of the underlying code (and I hope we will switch to `StreamingRead` for reading the data at some point in future, as well).
I will rename it to just `ReadMetadataOnly` (dropping the `Optimized`), should be less confusing.